ksql-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -465,6 +472,17 @@ private RowData(final long ts, final Struct key, final GenericRow value) {
     }
   }
 
+  private static class ProducerRecordInfo {
+    final RowData row;
+    final ProducerRecord<byte[], byte[]> record;
+
+    ProducerRecordInfo(final RowData row,
+        final ProducerRecord<byte[], byte[]> record) {
+      this.row = row;
```

Comment:
nit: null check.

(I know its an internal class - it just makes it easier to grok the code and know they are never null)